modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -259,6 +258,29 @@ const opPasteRowsAfter = function (grid: Structs.RowCells[], pasteDetails: Extra
   return outcome(mergedGrid, cursor);
 };
 
+const opGetColumnType = function (table: Element, target: TargetSelection): string {
+  const list = DetailsList.fromTable(table);
+  const house = Warehouse.generate(list);
```

Comment:
`const house = Warehouse.fromTable(table)` now that we have that helper function.